long_tests/torch_agent/ltest_dqn_vs_mdqn_acrobot.py

```diff
@@ -0,0 +1,95 @@
+from rlberry.envs import gym_make
+from rlberry.agents.torch import DQNAgent
+from rlberry.agents.torch import MunchausenDQNAgent as MDQNAgent
+from rlberry.manager import AgentManager, evaluate_agents, plot_writer_data
+import numpy as np
+import matplotlib.pyplot as plt
+import seaborn as sns
+
+
+def test_dqn_vs_mdqn_acro():
+
```

Comment:
Maybe add a docstring to say what result we should expect, or have an assert at the end to explain what result is good.